frontend/src/types.ts

```diff
@@ -449,16 +449,28 @@ export interface FunnelStepRangeEntityFilter extends EntityFilter {
 }
 
 export type EntityFilterTypes = EntityFilter | ActionFilter | FunnelStepRangeEntityFilter | null
-
 export interface PersonType {
```

Comment:
This comment got buried, but adding this under PersonType is a bad idea - we have PersonType used on a bunch of pages most of which have nothing to do with actors.

Reply:
Even more so since id in actors is the uuid, for persons it's the postgres id